Terminal.Gui/Drawing/LineCanvas.cs

```diff
@@ -413,13 +413,13 @@ private void ConfigurationManager_Applied (object? sender, ConfigurationManagerE
         bool useDouble = intersects.Any (i => i?.Line.Style == LineStyle.Double);
 
         bool useDashed = intersects.Any (
-                                         i => (i?.Line.Style == LineStyle.Dashed)
-                                              || (i?.Line.Style == LineStyle.RoundedDashed)
+                                         i => i?.Line.Style == LineStyle.Dashed
```

Comment:
Comparisons against possible nulls. These should be revisited at some point (probably very non-critical at the moment).